utils/fstack.c

```diff
@@ -1033,6 +1033,11 @@ bool fstack_check_opts(struct uftrace_task_reader *task, struct opts *opts)
 	if (opts->no_event && rec->type == UFTRACE_EVENT)
 		return false;
 
+	if (opts->no_sched && (rec->addr == EVENT_ID_PERF_SCHED_IN ||
+			       rec->addr == EVENT_ID_PERF_SCHED_OUT ||
+			       rec->addr == EVENT_ID_PERF_SCHED_BOTH))
```

Comment:
This is repeating.. you can consider adding a helper function like `is_sched_event(u64 addr)`.

Reply:
I made a helper `is_addr_sched_event` for this.